src/utils/time/index.ts

```diff
@@ -0,0 +1,11 @@
+export function getUNIXTime(dateTime: Date) {
```

Comment:
nit: I am not familiar with the coding guidelines for the project/framework but do we really need to name such utils file like  `src/utils/time/index.ts`?  Why not just name it like `src/utils/time.js`

Reply:
😅 don't know its coding guidelines, but have seen this pattern in many javascript projects.